test/queryRunner.test.ts

```diff
@@ -139,14 +174,34 @@ suite('Query Runner tests', () => {
             testQueryNotificationHandler.object,
             testVscodeWrapper.object
         );
+        queryRunner.batchSets[0] = {
+            executionElapsed: null,         // tslint:disable-line:no-null-keyword
```

Comment:
Yes, I am using nulls because null is what comes back from the service layer. Regardless of the "truthyness" of undefined vs null, I think for our testing scenarios it makes more sense to test with the same data that the service will send back than to use different data just because the tslinting rules say so.